### PR TITLE
Added Quickstart for Tomcat8 on App Engine Flexible.

### PR DESCRIPTION
**Pull request by kurtisvg**
This PR adds in a Quickstart for deploying an app on Tomcat8 on App Engine Flexible.

Due to the lack of a tomcat8-maven-plugin, it uses [Codehaus Cargo](https://codehaus-cargo.github.io/cargo/Maven2+plugin.html) to launch an embedded Tomcat8 with the app's war.

@lesv - FYI

### CODE REVIEW

**Comment by lesv** [comment]
@halgrimur FYI

File: flexible/tomcat/helloworld/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+* `gcloud components update`
+
+## Setup
+
+Use either:
+
+* `gcloud init`
+* `gcloud auth application-default login`
+
+We support building with [Maven](http://maven.apache.org/), [Gradle](https://gradle.org), and [IntelliJ IDEA](https://cloud.google.com/tools/intellij/docs/).
```
Comment (lesv) [inline flexible/tomcat/helloworld/README.md]: also eclipse

File: flexible/tomcat/helloworld/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+The samples have files to support both Maven and Gradle.  To use the IDE plugins, see the documentation pages above.
+
+## Maven
+[Using Maven and the App Engine Plugin](https://cloud.google.com/appengine/docs/flexible/java/using-maven)
+& [Maven Plugin Goals and Parameters](https://cloud.google.com/appengine/docs/flexible/java/maven-reference)
+### Running locally
```
Comment (lesv) [inline flexible/tomcat/helloworld/README.md]: newline before ###

File: flexible/tomcat/helloworld/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    $ mvn cargo:run
+  
+### Deploying
+
+    $ mvn appengine:deploy
+
+## Gradle
+[Using Gradle and the App Engine Plugin](https://cloud.google.com/appengine/docs/flexible/java/using-gradle)
```
Comment (lesv) [inline flexible/tomcat/helloworld/README.md]: newline before new para and after

File: flexible/tomcat/helloworld/eclipse-launch-profiles/AppEngineDeploy.launch [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (lesv) [inline flexible/tomcat/helloworld/eclipse-launch-profiles/AppEngineDeploy.launch]: rm -r eclipse-launch-profiles

File: flexible/tomcat/helloworld/nbactions.xml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lesv) [inline flexible/tomcat/helloworld/nbactions.xml]: rm nbactions.xml

File: flexible/tomcat/helloworld/src/main/appengine/app.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [END_EXCLUDE]
+
+# [START runtime]
+runtime: java
+env: flex
+
+handlers:
```
Comment (lesv) [inline flexible/tomcat/helloworld/src/main/appengine/app.yaml]: let's put the handlers last.

File: flexible/tomcat/helloworld/src/main/webapp/WEB-INF/web.xml [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<web-app xmlns="http://java.sun.com/xml/ns/javaee" version="3.1">
```
Comment (lesv) [inline flexible/tomcat/helloworld/src/main/webapp/WEB-INF/web.xml]: rm -r src/main/webapp/WEB-INF

**Comment by kurtisvg** [comment]
@lesv - Completed

File: flexible/tomcat/helloworld/gradle/wrapper/gradle-wrapper.properties [hunk 8]
```diff
@@ -0,0 +1,6 @@
+#Tue Jul 19 14:56:17 PDT 2016
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.13-bin.zip
```
Comment (lesv) [inline flexible/tomcat/helloworld/gradle/wrapper/gradle-wrapper.properties]: upgrade to a later version 4.xx

**Pull request merged**